cue/schemas/panels/stat/migrate.cue

```diff
@@ -27,6 +27,9 @@ if (*#panel.type | null) == "stat" {
 			}
 		}
 
-		// nothing to map to the `sparkline` field yet
+		#sparkline: *#panel.options.graphMode | "none"
+		if #sparkline == "area" {
+			sparkline: {}
+		}
 	}
-},
+}
```

Comment:
```suggestion
},
```

I know it's kinda counter-intuitive but the coma is actually mandatory here 😅 https://github.com/perses/perses/blob/main/docs/plugins/cue.md#panel-1